src/api/parseenums.py

```diff
@@ -133,19 +162,41 @@ def parse(self, filename):
                 continue
 
             if ENUM_END in line:
-                self.in_kinds = False
+                self.in_enum = False
                 break
-            elif self.in_kinds:
+            elif self.in_enum:
                 if line == OCB:
                     continue
-                name = line
+                name = None
+                value = None
                 if EQ in line:
-                    name = line[:line.find(EQ)].strip()
+                    (name, remainder) = line.split(EQ)
+                    name = name.strip()
+                    if C in remainder:
+                        value = int(remainder[:remainder.find(C)].strip())
+                    else:
+                        value = int(remainder)
                 elif C in line:
                     name = line[:line.find(C)].strip()
+                else:
+                    name = line
+
+                if not value:
+                    value = self.last_value + 1
+                self.last_value = value
+
+                enum = self.get_current_enum()
+                enum.enumerators[name] = value
                 fmt_comment = self.format_comment(self.latest_block_comment)
-                self.kinds_doc[name] = fmt_comment
+                enum.enumerators_doc[name] = fmt_comment
             elif ENUM_START in line:
-                self.in_kinds = True
+                self.in_enum = True
+                tokens = line.split(" ")
```

Comment:
```suggestion
                tokens = line.split()
```

This way split() does not produce empty strings in case of consecutive whitespaces.